community.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Community
+description: How to contribute to the Clowder community.
+background: '/PATH_TO_IMAGE'
```

Comment:
should this be here?

Reply:
It's a placeholder. I am trying to figure out what to put there to make the header more interesting and still readeable but I am not sure what. For some reason it doesn't complaint.